README.md

```diff
@@ -106,7 +106,12 @@ vSphere with Tanzu offers the vSAN Data Persistence platform. The platform provi
 
 - Using vSAN Data Persistence Platform (vDPP) with vSphere with Tanzu [documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-tanzu/GUID-F7223607-30A5-4B2D-9B06-A55A65FEAA11.html)
 - Enable Stateful Services in vSphere with Tanzu [documentation](https://docs.vmware.com/en/VMware-vSphere/7.0/vmware-vsphere-with-tanzu/GUID-F68B264E-76A3-4A6D-A3B0-17153DDF7A18.html)
-- Available vDPP Services [download](https://vmwaresaas.jfrog.io/ui/native/vDPP-Partner-YAML/)
+- Available vDPP Services 
+  - Couldian [1.2.0](https://vmwaresaas.jfrog.io/artifactory/vDPP-Partner-YAML/Cloudian/Hyperstore/SupervisorService/1.2.0/hyperstore-supervisorservice-1.2.0.yaml)
```

Comment:
typo: Cloudian

Reply:
Sorry my bad. Shouldn't have written the change in a rush. 

> Can you provide some description to the MR? The title says Minio, but it looks like the patch contains Cloudian updates too.

The Cloudian versions are not published this time. This change just puts them in different URL for download.